rpms/pulp-scl-python-django/pulp-scl-python-django.spec

```diff
@@ -0,0 +1,60 @@
+%{?scl:%scl_package python-django}
+%{!?scl:%global pkg_name %{name}}
+
+# Fix this build.  TODO: set this in pulp-scl properly
+%global python_version 3.5
+
+%global srcname Django
+
+Name:		%{?scl_prefix}python-django
+Version:	1.8.16
+Release:	7%{?dist}
+Summary:	A high-level Python Web framework
+
+Group:		Development/Languages
+License:	BSD
+URL:		http://www.djangoproject.com
+Source0:	https://files.pythonhosted.org/packages/source/D/Django/Django-%{version}.tar.gz
+
+BuildRequires:	%{?scl_prefix_python}python-devel
+BuildRequires:	%{?scl_prefix_python}python-setuptools
+%{?scl:BuildRequires:	%{scl}-build %{scl}-runtime}
+
+BuildRoot:      %{_tmppath}/%{pkg_name}-%{version}-%{release}-root-%(%{__id_u} -n)
+
+%{?scl:Requires:	%{scl}-runtime}
+
+%description
+Django is a high-level Python Web framework that encourages rapid
+development and a clean, pragmatic design. It focuses on automating as
+much as possible and adhering to the DRY (Don't Repeat Yourself)
+principle.
+
+
+%prep
+%setup -q -n %{srcname}-%{version}
+
+
+%build
+%{?scl:scl enable %{scl} "}
+%{__python} setup.py build
+%{?scl:"}
+
+
+%install
+%{?scl:scl enable %{scl} "}
```

Comment:
the trailing quote bothers me, but it works in the build. No changes requested, just complaining